constructor/nsis/main.nsi.tmpl

```diff
@@ -1139,7 +1139,7 @@ Section "Install"
     System::Call 'kernel32::SetEnvironmentVariable(t,t)i("CONDA_SOLVER", "classic").r0'
     SetDetailsPrint TextOnly
     DetailPrint "Checking virtual specs..."
-    push '"$INSTDIR\_conda.exe" create --dry-run --prefix "$INSTDIR" --offline @VIRTUAL_SPECS@'
+    push '"$INSTDIR\_conda.exe" create --dry-run --prefix "$INSTDIR\envs\_virtual_specs_checks" --offline @VIRTUAL_SPECS@'
```

Comment:
Do we need to create a temporary directory here, too?

Reply:
Micromamba is not used often on Windows due to the lack of menuinst, so it's not as critical. No clue how to create a tempdir via NSIS either.

Reply:
I think I know why we don't need to. We are setting `CONDA_PKGS_DIRS` here: https://github.com/conda/constructor/blob/main/constructor/nsis/main.nsi.tmpl#L1124

In the `sh` and `pkg` installers, that environment variable is set after the checks for virtual specs.